doc/man1/openssl-s_server.pod.in

```diff
@@ -656,7 +656,8 @@ affect the number of tickets sent after a resumption handshake.
 
 =item B<-serverpref>
 
-Use the server's cipher preferences, rather than the client's preferences.
+Use the server's preferences regarding ciphers, groups and signatures, rather
+than the client's preferences.
```

Comment:
This option essentially sets `SSL_OP_CIPHER_SERVER_PREFERENCE` which is poorly named. Perhaps we should have a new name for it (e.g. `SSL_OP_SERVER_PREFERENCE`, with `SSL_OP_CIPHER_SERVER_PREFERENCE` being an alias). Either way the documentation of the underlying option needs to be similarly updated:

https://github.com/openssl/openssl/blob/c1cd6d89a32d08d171b359aba0219357acf0c5cb/doc/man3/SSL_CTX_set_options.pod?plain=1#L125-L130

Reply:
> This option essentially sets SSL_OP_CIPHER_SERVER_PREFERENCE which is poorly named. 

Completely agree. For now, I chose to focus on getting documentation in line with real behaviour.

>  Perhaps we should have a new name for it (e.g. SSL_OP_SERVER_PREFERENCE, with SSL_OP_CIPHER_SERVER_PREFERENCE being an alias)

Please let me know if I should do this as part of this (originally only "simple" :) doc update or separately, @mattcaswell .

Reply:
> Please let me know if I should do this as part of this (originally only "simple" :) doc update or separately,

If you feel like tackling this problem then I'd do it in a separate PR.

But in this PR please also update the docs for `SSL_OP_CIPHER_SERVER_PREFERENCE` (on the SSL_CTX_set_options.pod page)

Reply:
So done in latest commit. Will try to do a separate PR introducing SSL_OP_SERVER_PREFERENCE in the code. I assume it'd be required to document SSL_OP_CIPHER_SERVER_PREFERENCE as a "deprecated" alias?! Any hints how to do this welcome.

Reply:
Changes eliminated following rebase. Please mark conversation resolved (pending separate PR as discussed).